frontend/pages/ManageControlsPage/ManageControlsPage.tsx

```diff
@@ -69,7 +72,9 @@ const ManageControlsPage = ({
 
   const navigateToNav = (i: number): void => {
     const navPath = controlsSubNav[i].pathname;
-    router.push(navPath);
+    const teamId = currentTeam?.id || undefined;
+    const queryString = teamId === undefined ? "" : `?team_id=${teamId}`;
+    router.replace(navPath + queryString);
```

Comment:
Just a heads up we use a function `{ buildQueryStringFromParams } from "utilities/url";` everywhere else in the app to add query strings to a url path. We should probably iterate after the release.

Reply:
cc: @lukeheath @ghernandez345

Reply:
Sorry I was in lunch and currently in a meeting or I'd hop on it